pkg/utils/utils.go

```diff
@@ -306,6 +306,7 @@ func IsFileExist(file string) bool {
 }
 
 // StringSliceEqual compare two string slice, ignore the order.
+// we also should consider if there has duplicate items in slice.
```

Comment:
I see here you are describing the case. But for fixing a bug, we should describe the functionality it provides. Maybe you can update the comment about when this function returns true precisely. @HusterWan

Reply:
updated